README.md

```diff
@@ -113,6 +112,36 @@ MyModel.myMethod = function(cb) {
   ctx.set('key', { foo: 'bar' });
 });
 ```
+### Bind for concurrency
```

Comment:
Please add an empty line before the section heading for consistency with the rest of this document.